.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,37 @@
+name: unit-tests
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-node@v1
+      with:
+          node-version: 10
```

Comment:
Can we make this the 12 LTS, or "latest lts"?

Reply:
Pending feature request at https://github.com/actions/setup-node/issues/26 -- for now I just set 12.

Reply:
SGTM